Header header
uint8 NumOfJoints
float64[] TurnDegree
int16[] VelocityRPM
uint8[] QuickStop
